subsystem/subsystem_suite_test.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 var db *gorm.DB
-var agentBMClient, badAgentBMClient, userBMClient, adminUserBMClient, unallowedUserBMClient *client.AssistedInstall
+var agentBMClient, badAgentBMClient, anotherAgentBMClient, userBMClient, adminUserBMClient, unallowedUserBMClient *client.AssistedInstall
```

Comment:
What does "another" mean in this context? I couldn't figure this one, the others are more or less clear (maybe except the "bad").

Reply:
I need to use a different `Agent` client to not interfere with other tests.
Any suggestion for a better name?

Reply:
by the scenario you're testing. e.g. failedSSOAgent

Reply:
Done